queries/1inch/use1InchQuoteQuery.ts

```diff
@@ -24,6 +24,10 @@ const use1InchQuoteQuery = (
 			network?.id!
 		),
 		async () => {
+			if (Number(amount) === 0) {
+				return '';
+			}
```

Comment:
Quoting for 0 is a returns a real (garbage) quote from 1Inch API so needed this to prevent that case